INFO     core.jenkins_utils:jenkins_utils.py:197 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:86 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 133
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:57 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:57 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:57 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:57 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:57 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:57 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 136.0.7103.92 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 136.0.7103.92 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 136.0.7103.92
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 136.0.7103.113
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 14109 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:36753
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "GET /session/cd664cc71c328e5c32b213ac2b533c0c/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/element/f.D3D15ADCAAEEBBA5662E3C63BABDC9C7.d.20CC7914EE160C3BDED82006A19064A0.e.1/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/element/f.D3D15ADCAAEEBBA5662E3C63BABDC9C7.d.20CC7914EE160C3BDED82006A19064A0.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/element/f.D3D15ADCAAEEBBA5662E3C63BABDC9C7.d.20CC7914EE160C3BDED82006A19064A0.e.9/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "GET /session/cd664cc71c328e5c32b213ac2b533c0c/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "GET /session/cd664cc71c328e5c32b213ac2b533c0c/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 156
DEBUG    core.jenkins_utils:jenkins_utils.py:26 update_crumb: 8d4303710eafd6f51d0582183b6d4f82b8ae9b70c84f7982f17a11a2ad221b5c
DEBUG    LoginPage:login_page.py:32 login crumb: 8d4303710eafd6f51d0582183b6d4f82b8ae9b70c84f7982f17a11a2ad221b5c
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "GET /session/cd664cc71c328e5c32b213ac2b533c0c/window HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "GET /session/cd664cc71c328e5c32b213ac2b533c0c/window/handles HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "GET /session/cd664cc71c328e5c32b213ac2b533c0c/window/handles HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/window HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "GET /session/cd664cc71c328e5c32b213ac2b533c0c/title HTTP/1.1" 200 0
DEBUG    Header:ui_element.py:56 Click on locator ('xpath', '//a[@href="/logout"]')
DEBUG    Header:ui_element.py:28 waiting for element_to_be_clickable(('xpath', '//a[@href="/logout"]'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "GET /session/cd664cc71c328e5c32b213ac2b533c0c/element/f.7A5ADE3A678962B15315EEC450546A02.d.526D4F4CB16AFB9517ED375B84871A09.e.75/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/element/f.7A5ADE3A678962B15315EEC450546A02.d.526D4F4CB16AFB9517ED375B84871A09.e.75/click HTTP/1.1" 200 0
DEBUG    LoginPage:ui_element.py:28 waiting for text_to_be_present_in_element(('xpath', '//main//h1'), 'Sign in to Jenkins')
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "GET /session/cd664cc71c328e5c32b213ac2b533c0c/element/f.7A5ADE3A678962B15315EEC450546A02.d.8EE76E9DC1DA138C4E0AE8D048A4AF1C.e.11/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/window HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "POST /session/cd664cc71c328e5c32b213ac2b533c0c/refresh HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "GET /session/cd664cc71c328e5c32b213ac2b533c0c/title HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36753 "DELETE /session/cd664cc71c328e5c32b213ac2b533c0c HTTP/1.1" 200 0